src/activities/morse_code/MorseCode.qml

```diff
@@ -370,12 +376,81 @@ ActivityBase {
 
         }
 
+        GCText {
+            id: morse_map
+            wrapMode: TextEdit.WordWrap
+            fontSize: mediumSize
+            horizontalAlignment: Text.AlignHCenter
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            width: parent.width * 0.8
+            height: parent.height * 0.8
+            visible: false
+            text: qsTr("\nA = .-\tB = -...\tC = -.-.\tD = -..\tE = .")
+                  + "\n" + qsTr("F = ..-.\tG = --.\tH = ....\tI = ..\t\tJ = .---")
+                  + "\n" + qsTr("K = -.-\tL = .-..\tM = --\tN = -.\tO = ---")
+                  + "\n" + qsTr("P = .--.\tQ = --.-\tR = .-.\tS = ...\tT = -")
+                  + "\n" + qsTr("U = ..-\tV = ...-\tW = .--\tX = -..-\tY = -.--\tZ = --..")
+                  + "\n\n" + qsTr("1 = .----\t2 = ..---\t3 = ...--\t4 = ....-\t5 = .....")
+                  + "\n" + qsTr("6 = -....\t7 = --...\t8 =---..\t9 = ----.\t0 = -----")
+
+            color: 'white'
+            Rectangle {
+                z: -1
+                opacity: 0.9
+                gradient: Gradient {
+                    GradientStop { position: 0.0; color: "#000" }
+                    GradientStop { position: 0.9; color: "#666" }
+                    GradientStop { position: 1.0; color: "#AAA" }
+                }
+                radius: 10
+                border.color: 'black'
+                border.width: 1
+                width: parent.width
+                height: parent.height * 0.9
+            }
+            Image {
```

Comment:
Depending on the resolution, the button is not visible because it is behind the one to display the map: http://pasteboard.co/mPLrZbZOO.png.
Same for the text (the Z letter in this case).